### DISCUSSION TITLE
problemi di installazione HACS su Hassos

### DISCUSSION THREAD

**Discussion by GiuseppeRus** [indomus/forum#2698] [HUB: Home Assistant]
Buongiorno,
ho installato hassos su mini pc, e sono  completamente incompetente in materia, tra i mille problemi incontrati questo proprio non riesco a risolverlo semplicemente cercando informazioni in giro.
Per l'installazione di hacs sto seguendo la guida di seguito https://indomus.it/guide/installare-hacs-home-assistant-community-store-sul-proprio-hub/.
quando copio il codice descritto sul terminal mi da sempre lo stesso errore:

 unable to resolve host address 'raw.githubusercontent.com'

vorrei anche provare la procedura manuale ma non so come raggiungere le cartelle descritte.
Grazie in anticipo per la risposta

**Comment by indomus** [comment]
> ##
> [<img alt="" width="32" height="32" src="https://avatars.githubusercontent.com/u/150027148?s=64&amp;v=4"> GiuseppeRus ](/GiuseppeRus) [ Nov 9, 2023 ](#discussion-5831358)
> -

`unable to resolve host address '[raw.githubusercontent.com](http://raw.githubusercontent.com/)'`

l'host ha palesi problemi a uscire su internet

**Reply by GiuseppeRus** [reply]
grazie per la risposta.
come posso fare per capire la natura del problema? è possibile che siano problemi legati al fornitore del servizio?

**Reply by snakuzzo** [reply]
Quale fornitore del servizio scusa ?
Il problema è da te...non riesci a risolvere a livello DNS quel dominio. Tutto qua.
Devi capire tu che problemi puoi avere.

**Reply by GiuseppeRus** [reply]
Ciao,
Il fornitore del servizio,la società che mi ha fornito il router con la connessione.
Scusami ma sono abbastanza ignorante in materia,  come posso capire dove e il problema?
Perdona la domanda  forse stupida ma ti riferisci al DNS del router?

**Reply by snakuzzo** [reply]
Più che altro a uelli che utilizza HA OS

**Reply by GiuseppeRus** [reply]
Per piacere potresti darmi qualche  Indicazione in più?

**Reply by snakuzzo** [reply]
vai da CLI e fai una lookup

es.

`nslookup raw.githubusercontent.com`

guarda...

![image](https://github.com/indomus/forum/assets/7205371/d4bd8204-e104-447d-abd4-3e434799400d)

**Reply by GiuseppeRus** [reply]
okay, a quanto pare ho qualcosa da studiare perchè non riesco a seguirti. Mi posteresti qualche link che mi aiuta a capire.

Grazie

**Reply by snakuzzo** [reply]
Devi usare la CLI di HA OS.  Se non l'hai già fatto, installa l'addon SSH & Web Terminal

**Reply by GiuseppeRus** [reply]
dopo ore e ore di ricerca sono riuscito a trovare con un pizzico di fortuna il comando per impostare i DNS

```
ha dns options --servers dns://8.8.8.8 --servers dns://8.8.4.4
```

grazie ai vostri input.

**Reply by snakuzzo** [reply]
bene. Ottimo!
